src/MUIDataTableBody.js

```diff
@@ -115,6 +122,8 @@ class MUIDataTableBody extends React.Component {
             <MUIDataTableBodyCell
               colSpan={options.selectableRows ? columns.length + 1 : columns.length}
               options={options}>
```

Comment:
this closing HTML bracket should be after the `rowIndex`

Reply:
Nice catch. Thank you!